pliers/extractors/models.py

```diff
@@ -62,12 +63,22 @@ class TFHubExtractor(Extractor):
 
     def __init__(self, url_or_path, features=None,
                  transform_out=None, transform_inp=None,
+                 load_type='keras', signature=None,
                  keras_kwargs=None):
         verify_dependencies(['tensorflow_hub'])
         if keras_kwargs is None:
             keras_kwargs = {}
         self.keras_kwargs = keras_kwargs
-        self.model = hub.KerasLayer(url_or_path, **keras_kwargs)
+        self.load_type = load_type
+
+        if load_type == 'keras':
+            self.model = hub.KerasLayer(url_or_path, **keras_kwargs)
+        elif load_type == 'saved_model':
+            model = hub.load(url_or_path)
+            if signature is None:
+                signature = list(model.signatures.keys())[0]
+            self.model = model.signatures[signature]
```

Comment:
Ah, good to know. I tried setting just `signature` and it didnt work. If what you say is true then I might prefer to just support one, since the user shouldn't care too much which one it is.

Reply:
Oh hey, you're right! Let's stick to one way of loading then, and we can document how to use SPICE using tests / docs